Firestore/Source/Local/FSTLevelDBMigrations.h

```diff
@@ -29,12 +30,13 @@ typedef int32_t FSTLevelDBSchemaVersion;
 /**
  * Returns the current version of the schema for the given database
  */
-+ (FSTLevelDBSchemaVersion)schemaVersionForDB:(std::shared_ptr<leveldb::DB>)db;
++ (FSTLevelDBSchemaVersion)schemaVersion:
```

Comment:
`schemaVersionWithTransaction` would be more ObjC-style conforming.

Reply:
Done.